Dockerfile

```diff
@@ -17,54 +17,55 @@ LABEL org.opencontainers.image.source https://github.com/helmfile/helmfile
 
 RUN apk add --no-cache ca-certificates git bash curl jq openssh-client
 
-ARG HELM_VERSION="v3.10.2"
```

Comment:
why use `ENV`?

Reply:
I want to keep this open for overriding at build time. Could you revert this line? @jouve

Reply:
sure (it was just for consistency with the other tools)

Reply:
I propose

```
ARG HELM_VERSION
ENV HELM_VERSION=$HELM_VERSION
```

so that the version of tools installed can be easily get with one-liner

```
env | grep '_VERSION$'
```